### DISCUSSION TITLE
Requirements.txt file is missing

### DISCUSSION THREAD

**Discussion by sampath1976** [SWivid/F5-TTS#283] [General]
Dear All, I tried to install this with the steps and now when I see YouTube demos, all are saying we need to install requirements.txt. but currently its not available in GitHub. Please help.

**Comment by SWivid** [comment]
@sampath1976 just follow the instructment in README.md. It's all easy now~

**Reply by spike4379** [reply]
I agree it is nicer now, BUT having the requirements.txt there is nice for making custom bat installations.
Better to have and not need, than need and not have :)

**Reply by SWivid** [reply]
Sure, but it is more common for recent projects to use `pyproject.toml` instead of `setup.py` and `requirements.txt`.
The previous command `pip install -r requirements.txt` is the `pip install -e .` for now, which is also possible to do it in bat (the dependencies are also listed inside).
